meson.build

```diff
@@ -487,7 +487,7 @@ install_headers(sdb_inc_files, install_dir: rizin_incdir / 'sdb')
 librzsdb = static_library('rzsdb', sdb_files,
   include_directories: sdb_inc,
   implicit_include_directories: false,
-  c_args: host_machine.system() == 'windows' ? '-DSDB_API=__declspec(dllexport)' : [],
+  c_args: host_machine.system() == 'windows' ? '-DSDB_IPI= -DSDB_API=__declspec(dllexport)' : [],
```

Comment:
I think there is no need for `SDB_API=__declspec(dllexport)` because everything is already defined in https://github.com/rizinorg/sdb/blob/master/src/types.h#L16 . And actually, that part IMHO should be rewritten according to https://gcc.gnu.org/wiki/Visibility .

Reply:
And if you remove `SDB_API=` also the `SDB_IPI=` should be removed ofc.

Reply:
@ret2libc isn't this `visibility` attribute just GCC/Clang-specific? Declspec is the one that is supported by MSVC.

Reply:
nah, there is everything there for both gcc/clang and msvc.

Reply:
I just removed the whole line and it works as is.